lib/geordi/settings.rb

```diff
@@ -93,7 +91,11 @@ def check_for_invalid_keys(settings, allowed_keys, file)
         Geordi::Interaction.warn "Geordi detected unknown keys in #{file}.\n"
 
         invalid_keys.sort.each do |key|
-          puts "* #{key}"
+          if key == :use_vnc
+            puts '* use_vnc: Unsupported setting "use_vnc". Please remove this setting from /home/user/projects/the-project/.geordi.yml.'
```

Comment:
Please replace the path with the real path of the settings, so the user knows where she needs to delete the file.